src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/AddDraftApprenticeshipViewModel.cs

```diff
@@ -1,11 +1,14 @@
 using System;
 using System.ComponentModel.DataAnnotations;
+using FluentValidation.Attributes;
 using SFA.DAS.ProviderCommitments.Domain_Models.ApprenticeshipCourse;
 using SFA.DAS.ProviderCommitments.ModelBinding.Models;
 using SFA.DAS.ProviderCommitments.Models;
+using SFA.DAS.ProviderCommitments.Web.Requests;
 
 namespace SFA.DAS.ProviderCommitments.Web.Models
 {
+    [Validator(typeof(AddDraftApprenticeshipViewModelValidator))]
```

Comment:
We don't need this attribute - we've injected FV into the MVC pipeline.

Reply:
Good point

Reply:
addressed and removed duplicate IMediator registration from DefaultRegistry